[2024-11-11T21:14:56.775+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-11T21:14:56.799+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: two_stocks_DAG.train_stock_model__1 manual__2024-11-11T21:09:48.979962+00:00 [queued]>
[2024-11-11T21:14:56.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: two_stocks_DAG.train_stock_model__1 manual__2024-11-11T21:09:48.979962+00:00 [queued]>
[2024-11-11T21:14:56.807+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2024-11-11T21:14:56.822+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): train_stock_model__1> on 2024-11-11 21:09:48.979962+00:00
[2024-11-11T21:14:56.829+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=556) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-11T21:14:56.831+0000] {standard_task_runner.py:63} INFO - Started process 561 to run task
[2024-11-11T21:14:56.830+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'two_stocks_DAG', 'train_stock_model__1', 'manual__2024-11-11T21:09:48.979962+00:00', '--job-id', '194', '--raw', '--subdir', 'DAGS_FOLDER/two_stocks.py', '--cfg-path', '/tmp/tmpk5cg5qnt']
[2024-11-11T21:14:56.832+0000] {standard_task_runner.py:91} INFO - Job 194: Subtask train_stock_model__1
[2024-11-11T21:14:56.880+0000] {task_command.py:426} INFO - Running <TaskInstance: two_stocks_DAG.train_stock_model__1 manual__2024-11-11T21:09:48.979962+00:00 [running]> on host 889c1536fdc4
[2024-11-11T21:14:56.972+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='two_stocks_DAG' AIRFLOW_CTX_TASK_ID='train_stock_model__1' AIRFLOW_CTX_EXECUTION_DATE='2024-11-11T21:09:48.979962+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-11T21:09:48.979962+00:00'
[2024-11-11T21:14:56.974+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-11T21:14:56.984+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-11T21:14:56.985+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-11T21:14:56.987+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-11T21:14:57.782+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-11T21:14:57.953+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-11T21:14:58.457+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-11T21:16:04.967+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-11T21:16:06.680+0000] {cursor.py:1149} INFO - Number of results in first chunk: 7
[2024-11-11T21:16:06.681+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-11T21:16:06.682+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-11T21:16:06.690+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=two_stocks_DAG, task_id=train_stock_model__1, run_id=manual__2024-11-11T21:09:48.979962+00:00, execution_date=20241111T210948, start_date=20241111T211456, end_date=20241111T211606
[2024-11-11T21:16:06.712+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-11T21:16:06.726+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-11T21:16:06.728+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
